pkg/operator/deploy/deploy.go

```diff
@@ -49,13 +49,13 @@ type operator struct {
 	env env.Interface
 	oc  *api.OpenShiftCluster
 
-	dh            dynamichelper.Interface
-	cli           kubernetes.Interface
-	extensionscli extensionsclient.Interface
 	arocli        aroclient.Interface
+	extensionscli extensionsclient.Interface
+	kubernetescli kubernetes.Interface
+	dh            dynamichelper.Interface
```

Comment:
New standard for clientsets is alphabetization, except for `dh`, which is similar enough to the other Interfaces to be in the same field grouping, but different enough to always appear at the end.